packages/fxa-payments-server/src/components/Survey/index.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import MockApp from '../../../.storybook/components/MockApp';
+import Survey from './index';
+
+const surveyURL = 'https://www.surveygizmo.com/s3/5541940/pizza';
+
+storiesOf('components/Survey', module)
+  .add('default', () => (
+    <MockApp>
+      <Survey surveyURL={surveyURL} />
+    </MockApp>
+  ))
+  .add('complete', () => (
+    <MockApp>
+      <Survey surveyURL={surveyURL} surveyComplete={true} />
```

Comment:
This can be written as `<Survey {...{surveyURL}} surveyComplete />` as well as line 31 in the test file, but I get wanting to explicitly set `surveyComplete` to `true`. Wonder if we can set up a linter warning to stick a yellow squiggly under `surveyURL={surveyURL}`, or maybe the team doesn't actually like that shorthand, I don't see it used much. (`surveyURL` can be shortened on line 11 as well)